app/views/layouts/application.html.erb

```diff
@@ -10,14 +10,46 @@
   </head>
 
   <body>
-    <% if notice.present? %>
-      <p class="alert alert-info"><%= notice %></p>
-    <% end %>
 
-    <% if alert.present? %>
-      <p class="alert alert-danger"><%= alert %></p>
-    <% end %>
+     <nav class="navbar navbar-toggleable-md navbar-inverse bg-inverse">
+       <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
+         <span class="navbar-toggler-icon"></span>
+       </button>
 
-    <%= yield %>
-  </body>
-</html>
\ No newline at end of file
+       <div class:="class:" 'sm-3'>
```

Comment:
Check this line out. Is is supposed to be `<div class="sm-3">`?

Reply:
fixed,good catch